Project.toml

```diff
@@ -13,7 +13,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-DataFrames = "0.17, 0.18, 0.19, 0.20"
+DataFrames = "0.19, 0.20, 0.21"
```

Comment:
> Note DataFrames 0.18 and 0.19 do not support the new ! syntax.

Should we remove `0.19` here?

Reply:
I'm not sure why I wrote that. DataFrames 0.19 does support the `!` syntax and the tests do work against 0.19